WordPress/Classes/ViewRelated/Post/PostMediaUploadViewModel.swift

```diff
@@ -14,6 +15,11 @@ final class PostMediaUploadViewModel: ObservableObject {
     private let post: AbstractPost
     private let coordinator: MediaCoordinator
     private weak var timer: Timer?
+    private var cancellables: [AnyCancellable] = []
+
+    var isButtonRetryDisabled: Bool {
+        !(uploads.contains(where: { $0.error != nil }))
```

Comment:
It's enabled by design so that you could retry the uploads if you do something to fix an error. For example, you could clear some space in the media library.

Reply:
Sounds good 👍